tools/msccl-algorithms/allreduce-allpairs-8n-Simple-1pass-op.xml

```diff
@@ -0,0 +1,2178 @@
+<algo name="allreduce_pairs" proto="Simple" nchannels="2" nchunksperloop="16" ngpus="8" coll="allreduce" inplace="0" outofplace="1" minBytes="0" maxBytes="25599">
```

Comment:
We are supposed to use LL protocol for such small message sizes. Does the new xml provider better performance for all configs (1 GPU per process, etc.)?

Reply:
The performance is unchanged (more or less the same) compared to LL for 1 GPU per thread. I can test it our with 8 MPI processes

Reply:
Comparison is in this comment https://ontrack-internal.amd.com/browse/LWPCOMMLIBS-422#:~:text=I-,resolved,-the%20noticeable%20degradation

Reply:
I think we should try to understand where the regression coming from before using workaround. Switching LL to Simple seems to be dramatic measure.

Reply:
If you run it a few times, LL also results in a validation error with `-g 8 -t 1 -b4 -e 1M -f 2`. This has to be investigated as a defect.